src/website/views/components/accordion/_examples.pug

```diff
@@ -0,0 +1,261 @@
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+
+      .chi-accordion
+        .chi-accordion__header
+          .chi-accordion__title
+            i.chi-icon.icon-chevron-down.-mr--2
+            | Accordion 3
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+
+      .chi-accordion
+        .chi-accordion__header
+          .chi-accordion__title
+            i.chi-icon.icon-chevron-down.-mr--2
+            | Accordion 4
+        .chi-accordion__content
+          p.chi-accordion__text Content goes here
+  .example-tabs.-pl--2
+    ul#chi-tabs-e1.chi-tabs
+      li.-disabled
+        a(href='#chi-tabs-e1-web') Web Component
+      li.-active
+        a(href='#chi-tabs-e1-html') HTML Blueprint
+  #chi-tabs-e1-web.chi-tabs-panel
+  #chi-tabs-e1-html.chi-tabs-panel.-active
+    .chi-tab__description
+      span.-text--grey
+        | This HTML Blueprint requires JavaScript.
+        | You may use your own solution, or use Chi's vanilla JavaScript solution,
+        | <a href="../../getting-started/installation">Chi.js</a>.
+    :code(lang="html")
+      <div class="chi-accordion">
+          <div class="chi-accordion__header">
+              <div class="chi-accordion__title"><i class="chi-icon icon-chevron-down -mr--2"></i>Accordion 1</div>
+          </div>
+          <div class="chi-accordion__content">
+              <p class="chi-accordion__text">Content goes here</p>
+          </div>
+      </div>
+      <div class="chi-accordion -expanded">
+          <div class="chi-accordion__header">
+              <div class="chi-accordion__title"><i class="chi-icon icon-chevron-up -mr--2"></i>Accordion 2</div>
+          </div>
+          <div class="chi-accordion__content">
+              <p class="chi-accordion__text">Content goes here</p>
+          </div>
+      </div>
+      <div class="chi-accordion">
+          <div class="chi-accordion__header">
+              <div class="chi-accordion__title"><i class="chi-icon icon-chevron-down -mr--2"></i>Accordion 3</div>
+          </div>
+          <div class="chi-accordion__content">
+              <p class="chi-accordion__text">Content goes here</p>
+          </div>
+      </div>
+      <div class="chi-accordion">
+          <div class="chi-accordion__header">
+              <div class="chi-accordion__title"><i class="chi-icon icon-chevron-down -mr--2"></i>Accordion 4</div>
+          </div>
+          <div class="chi-accordion__content">
+              <p class="chi-accordion__text">Content goes here</p>
+          </div>
+      </div>
+
+      <!-- JavaScript -->
+      <script>
+      chi.accordion(
+        document.querySelectorAll('.chi-tabs-panel .chi-tabs'));
+      </script>
+
+h3 Portal
+.example.-mb--3#example2
+  .-p--3.-p-lg--6.-bg--grey-10
+    .-mw--720.-mx--auto
+      .chi-accordion.-portal.-bordered
+        .chi-accordion__header
+          .chi-accordion__title
+            i.chi-icon.icon-plus.-mr--2
```

Comment:
Let's change all expand/collapse icons in the portal example to chevrons to match the base accordion example.